apps/web/src/components/Basenames/RenewalFlow.tsx

```diff
@@ -0,0 +1,129 @@
+'use client';
+import { Transition } from '@headlessui/react';
+import RegistrationBackground from 'apps/web/src/components/Basenames/RegistrationBackground';
+import RenewalForm from 'apps/web/src/components/Basenames/RenewalForm';
+import { UsernamePill } from 'apps/web/src/components/Basenames/UsernamePill';
+import { UsernamePillVariants } from 'apps/web/src/components/Basenames/UsernamePill/types';
+import useBasenameChain, { supportedChainIds } from 'apps/web/src/hooks/useBasenameChain';
+import classNames from 'classnames';
+import { useCallback, useEffect, useMemo } from 'react';
+import { useAccount, useSwitchChain } from 'wagmi';
+import { Basename } from '@coinbase/onchainkit/identity';
+import { Icon } from 'apps/web/src/components/Icon/Icon';
+import { useRouter } from 'next/navigation';
+import { RegistrationSteps } from 'apps/web/src/components/Basenames/RegistrationContext';
+
+type RenewalFlowProps = {
+  name: string;
+};
+
+export function RenewalFlow({ name }: RenewalFlowProps) {
+  const { chain } = useAccount();
+  const { basenameChain } = useBasenameChain();
+  const { switchChain } = useSwitchChain();
+  const router = useRouter();
+
+  const isOnSupportedNetwork = useMemo(
+    () => chain && supportedChainIds.includes(chain.id),
+    [chain],
+  );
+
+  const switchToIntendedNetwork = useCallback(
+    () => switchChain({ chainId: basenameChain.id }),
+    [basenameChain.id, switchChain],
+  );
+
+  const onBackArrowClick = useCallback(() => {
```

Comment:
this doesn't seem like a great choice, if a user clicks this button after just landing on the page it will send them to some other site. 

are there even steps in this flow? can we nix the back arrow entirely?